frontend/components/App/App.tsx

```diff
@@ -28,9 +27,11 @@ import Spinner from "components/Spinner";
 interface IAppProps {
   children: JSX.Element;
   router: InjectedRouter;
-  location?: {
-    pathname: string;
-  };
+  location:
+    | {
+        pathname: string;
+      }
+    | undefined;
 }
 
 const App = ({ children, location, router }: IAppProps): JSX.Element => {
```

Comment:
We're not using the `router` prop anymore, but I left it in so we'd know it was available.